src/CMake/dkms-azure.cmake

```diff
@@ -7,19 +7,19 @@ set(DKMS_POSTINST "postinst-azure")
 set(DKMS_PRERM "prerm-azure")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`